tiflash: update behavior under CBO mode (#2405)
pingcap/docs: reference/tiflash/use-tiflash.md

In the result of above statement:

> * `AVAILABLE` indicates whether the TiFlash replicas of this table is available or not. `1` means available and `0` means unavailable. Once the replicas become available, this status does not change. If you have used DDL statements to modify the number of replicas, the replication status is recalculated.

Reviewer: ```suggestion
* `AVAILABLE` indicates whether the TiFlash replicas of this table is available or not. `1` means available and `0` means unavailable. Once the replicas become available, this status does not change. If you use DDL statements to modify the number of replicas, the replication status will be recalculated.
```

`cop[tiflash]` means that the task will be sent to TiFlash for processing. If you have not selected a TiFlash replica, you can try to update the statistics using the `analyze table` statement, and then check the result using the `explain analyze` statement.

> Note that if a table has only a single TiFlash replica and the node cannot provide service, queries in the smart mode will repeatedly retry. In this situation, you need to specify the engine or use the manual Hint to read data from TiKV.

Reviewer: ```suggestion
Note that if a table has only a single TiFlash replica and the related node cannot provide service, queries in the CBO mode will repeatedly retry. In this situation, you need to specify the engine or use the manual Hint to read data from TiKV.
```
See https://github.com/pingcap/docs-cn/pull/2883#pullrequestreview-400490430 for my suggestion.
